### ISSUE TITLE
About "attention_vectors = torch.cat([attention_vectors, vector], dim=1)"

### ISSUE THREAD

**Issue by XiaoAHeng** [pppLang/SKNet#8]
In your code,`feas = torch.cat([feas, fea], dim=1);torch.cat([attention_vectors, vector], dim=1)`, however, in the original paper,V=a*U1+b*U2, if "dim" in th.cat([attention_vectors, vector]) is 1, I think the "dim" is 0,am I right?

**Comment by ascenoputing** [comment]
I'd rather join you in the doubt

**Comment by ascenoputing** [comment]
It seems that the SoftMax(dim=1) should be SoftMax(dim=0)
